ql/src/test/org/apache/hadoop/hive/ql/TestTxnCommands3.java

```diff
@@ -197,10 +193,10 @@ private void testDeleteEventPruning() throws Exception {
 
     String[][] expected2 = new String[][]{
         {"{\"writeid\":1,\"bucketid\":536870912,\"rowid\":1}\t4\t5",
-            "warehouse/t/base_0000001/bucket_00000"},
+            "warehouse/t/base_0000003_v0000012/bucket_00000"},
```

Comment:
It was never `base_0000001` even with MR but this was never asserted because this test always runs in vectorized mode so `INPUT__FILE__NAME` was not queried.
I altered the test query by adding `INPUT__FILE__NAME` since it is no longer a show stopper for vectorization and I had to change the expected base directory name accordingly.

Reply:
👍